lottie/src/main/java/com/airbnb/lottie/LottieDrawable.java

```diff
@@ -406,7 +406,9 @@ public void run(LottieComposition composition) {
       });
       return;
     }
-    animator.resumeAnimation();
+    if (systemAnimationsEnabled) {
```

Comment:
In playAnimation() call, it checks 2 conditions like:
`if (systemAnimationsEnabled || getRepeatCount() == 0) {
      animator.playAnimation();
    }`
Do we need to do so here? Not get why need to check getRepeatCount() == 0 in playAnimation.